app/Models/MiscModel.php

```diff
@@ -345,6 +345,17 @@ public function menuItems(array $items = []): array
             ];
         }
 
+        // Permissions for the admin?
+        if (auth()->check() && $this->entity->mentionsCount() > 0) {
+            $items['fourth']['mentions'] = [
+                'name' => 'crud.tabs.mentions',
+                'route' => 'entities.mentions',
+                'entity' => true,
+                'count' => $this->entity->mentionsCount(),
```

Comment:
I might be wrong but this looks like it's doing a complex query to the database twice on every page load, so the count needs to be cached.